evap/results/templates/results_index.html

```diff
@@ -44,21 +98,23 @@ <h3>{% trans 'Evaluation results' %}</h3>
                                 <span class="course-name">{{ course.name }}</span>
                                 <div>
                                     {% for degree in course.degrees.all %}
-                                        <span class="badge badge-primary">{{ degree }}</span>
+                                        <span class="badge badge-primary badge-degree">{{ degree }}</span>
```

Comment:
some of these classes are used for styling, and another, similarly-looking class is used for identification, is that the way this is usually done?

Reply:
Usually one would like to use an ID for identification but those should be unique and it doesn't really fit here. A class is a good solution to my mind because it's actually quite similar to what we do for styling. There we define that all objects of a certain class look like this and that and here we're using the class selector to get all objects with this class to collect their data. It might be better to only use the class `badge-degree` and let the css map its looks to what is defined in `badge-primary`, but I'd say that's not necessary. So - I would continue using it in this way.